pkg/api/v1alpha2/types_include_config.go

```diff
@@ -57,10 +66,18 @@ func (ic *IncludeConfig) ConvertToDiffIncludeConfig() (dic action.DiffIncludeCon
 
 		dpkg := action.DiffIncludePackage{Name: pkg.Name}
 		switch {
-		case !pkg.StartingVersion.EQ(semver.Version{}):
-			dpkg.Versions = []semver.Version{pkg.StartingVersion}
-		case pkg.StartingBundle != "":
-			dpkg.Bundles = []string{pkg.StartingBundle}
+		case pkg.MinVersion != "" && pkg.MaxVersion != "":
```

Comment:
This is a good point. I would like to understand how we could accomplish this while still having uniformity throughout the configuration. We went with `minVersion` and `maxVersion` to give a consistent experience with the way `releases` are specified. The reason we have `minVersion` and `maxVersion` for releases is that we allow mirroring with the ranges or a `shortestPath` calculation between the two values. @afflom Do you have any thoughts on this and whether the `shortestPath` option is worth blocking a potentially more user-friendly way of specifying the ranges? Maybe we can simplify this more by removing that field in `releases`.

Reply:
At the same time, my concern with allowing `ranges` to be specified directly is the flexibility and possibly unexpected versions that may be missing from the mirrored.

Reply:
I wanted to bring up this point because it is something that is possible with the new changes in `diff`. However, it doesn't have to be something we have to do now for this release. It is possible to have subsequent PRs to offer this option in the next release or next week if we really want to offer this option for this release. My intention is still to merge this PR as it is.

Reply:
I understand. Thanks for bringing it up! I definitely think it is something we should discuss.